crates/sparrow-runtime/src/execute/compute_executor.rs

```diff
@@ -245,7 +243,6 @@ impl ComputeExecutor {
 
                 Ok(ProgressUpdate::ExecutionComplete { compute_snapshots })
             };
-
```

Comment:
Looks like a lot of unrelated changes -- removing empty lines? Probably don't want to include those changes.